spec/helpers/locale_helper_spec.rb

```diff
@@ -0,0 +1,46 @@
+RSpec.describe LocaleHelper do
```

Comment:
Nitpick: This should be added in the same commit as `LocaleHelper`. Perhaps they can be added together in their own commit? It would make the audit trail easier to follow.

Reply:
Now added in same commit.